docs/usage/configuration-options.md

```diff
@@ -1655,6 +1655,27 @@ Under the hood, it creates a MR-level approval rule where `approvals_required` i
 This option works only when `automerge=true` and either `automergeType=pr` or `automergeType=branch`.
 Also, approval rules overriding should not be [prevented in GitLab settings](https://docs.gitlab.com/ee/user/project/merge_requests/approvals/settings.html#prevent-editing-approval-rules-in-merge-requests).
 
+## gitLabReviewersFromApprovalRule
+
+There are cases where you want to automatically add reviewers to Renovate PRs based on existing GitLab approval rules (<https://docs.gitlab.com/user/project/merge_requests/approvals/rules/>)
+Renovate can use a GitLab's project approval rule to determine who should review dependency updates.
+This is particularly useful for teams that already have well-defined approval processes in GitLab.
+
+To enable this functionality, use the gitLabReviewersFromApprovalRule option to specify the name of the GitLab approval rule to use for selecting reviewers.
+Renovate will then add all reviewers from that rule to the created merge requests. It will add the reviewers only at time of MR creation and does not update them after that.
```

Comment:
```suggestion
Renovate will then add all reviewers from that rule to merge requests it creates.
It will add the reviewers only at time of MR creation and does not update them after that.
```